plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/KtlintPlugin.kt

```diff
@@ -99,7 +100,13 @@ open class KtlintPlugin : Plugin<Project> {
                 }
 
                 variantManager?.variantScopes?.forEach {
-                    val kotlinSourceDir = target.files(it.variantData.javaSources)
+                    val sourceDirs = it.variantData.javaSources
+                            .fold(mutableListOf<File>()) { acc, configurableFileTree ->
+                                acc.add(configurableFileTree.dir)
+                                acc
```

Comment:
Could be replaced with `acc += configurableFileTree.dir` right?

Reply:
Actually no, because `plusAssign` fun returns `Unit`, so additional `acc` still required.

Reply:
Fair enough.